maven_iuvs/echelle.py

```diff
@@ -201,26 +201,34 @@ def downselect_data(light_index, orbit=None, date=None, segment=None, lat=None,
 
```

Comment:
did you want to add comments for the new arguments?

Reply:
Yes, now here: https://github.com/lasp/maven_iuvs/pull/67/commits/f006aea93e7b09ffb4882f8f7ba716aa8ec80b5b